tasks/winbuildscripts/unittests.ps1

```diff
@@ -81,7 +81,7 @@ if($err -ne 0){
 }
 
 & inv -e install-tools
-& inv -e test --skip-linters --junit-tar="$Env:JUNIT_TAR" --race --profile --rerun-fails=2 --cpus 8 --arch $archflag --python-runtimes="$Env:PY_RUNTIMES" --python-home-2=$Env:Python2_ROOT_DIR --python-home-3=$Env:Python3_ROOT_DIR --save-result-json C:\mnt\test_output.json $Env:EXTRAPARAMS
+& inv -e test --skip-linters --junit-tar="$Env:JUNIT_TAR" --race --profile --rerun-fails=2 --cpus 8 --arch $archflag --python-runtimes="$Env:PY_RUNTIMES" --python-home-2=$Env:Python2_ROOT_DIR --python-home-3=$Env:Python3_ROOT_DIR --save-result-json C:\mnt\$Env:TEST_OUTPUT_FILE $Env:EXTRA_OPTS
```

Comment:
can you give a default value to `Env:TEST_OUTPUT_FILE` so the script can still be run outside of CI without having to manually set this var?

Reply:
Did not take into account that it was used outside the CI. Added a default value!